lib/download.js

```diff
@@ -12,76 +12,15 @@ const fsCopyFile = util.promisify(fs.copyFile);
 const fsMkdir = util.promisify(fs.mkdir);
 const fsReadDir = util.promisify(fs.readdir);
 
-/**
- * @param {string[]} assets
- * @param {{ 'win32'?: ('x64' | 'ia32' | 'arm64')[]; 'linux'?: ('arm' | 'x64' | 'arm64' | 'armhf')[]; 'darwin'?: ('x64' | 'arm64')[]; 'alpine'?: ('x64' | 'arm64')[] }} platformOptions If not provided, then downloads all binaries for all platforms and archs.
- * @return {string[]} List of assets to copy
- */
-function getAssetsToCopy(assets, platformOptions) {
```

Comment:
In the package, end users will no longer download files from github, hence this is dead code